lib/features/personal_summary/pages/personal_summary_page.dart

```diff
@@ -0,0 +1,440 @@
+          icon: Icon(
+            left ? Icons.arrow_back_ios : Icons.arrow_forward_ios,
+            color: AppTheme.givtBlue,
+            size: 17,
+          ),
+        ),
+      );
+
+  Widget _buildNarrowWidget({
+    required Size size,
+    required AppLocalizations locals,
+    required bool left,
+    required String countryCharacter,
+    required PersonalSummaryState state,
+  }) =>
+      Container(
+        decoration: BoxDecoration(
+          color: left ? AppTheme.givtLightGreen : Colors.white,
+          border: Border.all(color: Colors.transparent),
+          borderRadius: const BorderRadius.all(Radius.circular(10)),
+          boxShadow: const [
+            BoxShadow(
+              color: Colors.grey,
+              offset: Offset(0, 5),
+              blurRadius: 10,
+            ),
+          ],
+        ),
+        child: SizedBox(
+          height: 150,
+          width: left ? size.width * 0.32 : size.width * 0.4,
+          child: left
+              ? Column(
+                  children: [
+                    const Spacer(),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        Text(
+                          '$countryCharacter ${state.monthlyGivts.fold<double>(0, (sum, item) => sum + item.amount)}',
+                          style: const TextStyle(
+                            fontSize: 24,
+                            color: Colors.white,
+                            fontWeight: FontWeight.w900,
+                          ),
+                        ),
+                      ],
+                    ),
+                    const Spacer(),
+                    Padding(
+                      padding: const EdgeInsets.all(8),
+                      child: Text(
+                        locals.budgetSummaryBalance,
+                        textAlign: TextAlign.center,
+                        style: const TextStyle(
+                          fontSize: 14,
+                          color: Colors.white,
+                        ),
+                      ),
+                    ),
+                    const Spacer(),
+                  ],
+                )
+              : Column(
+                  children: [
+                    const Spacer(),
+                    Image.asset(
+                      'assets/images/givy_money.png',
+                      height: 60,
+                    ),
+                    const Spacer(
+                      flex: 2,
+                    ),
+                    Text(
+                      locals.budgetSummarySetGoalBold,
+                      style: const TextStyle(
+                        fontSize: 14,
+                        fontWeight: FontWeight.w900,
+                      ),
+                    ),
+                    const Text(
+                      //locals.budgetSummarySetGoal,
+                      'Giving goal feature coming soon!',
```

Comment:
The commented out one is the correct string for that widget, but since the button is inactive i wrote that temporarily to not confuse users. As soon as its implemented will be changed to the l10n string.